contrib/platform/src/com/sun/jna/platform/win32/Kernel32.java

```diff
@@ -3289,4 +3289,39 @@ boolean GetVolumePathNamesForVolumeName(String lpszVolumeName,
      *         information, call GetLastError.
      */
     boolean EnumResourceNames(HMODULE hModule, Pointer type, WinBase.EnumResNameProc proc, Pointer lParam);
+    
+    /**
+     * Retrieves information about the first module associated with a process.
+     * 
+     * @see <a href="https://msdn.microsoft.com/en-us/library/windows/desktop/ms684218(v=vs.85).aspx">MSDN</a>
+     * @param hSnapshot
+     *            A handle to the snapshot returned from a previous call to the
+     *            CreateToolhelp32Snapshot function.
+     * @param lpme
+     *            A pointer to a MODULEENTRY32 structure.
+     * @return Returns TRUE if the first entry of the module list has been
+     *         copied to the buffer or FALSE otherwise.<br>
+     *         The ERROR_NO_MORE_FILES error value is returned by the
+     *         GetLastError function if no modules exist or the snapshot does
+     *         not contain module information.
+     */
+    boolean Module32First(HANDLE hSnapshot, Tlhelp32.MODULEENTRY32.ByReference lpme);
```

Comment:
Because this depends on a `Structure` with explicit native wide strings, you should explicitly map to `Module32FirstW`.  I believe you should still be able to reference with `Module32First` in client code.